odl/space/cu_ntuples.py

```diff
@@ -1453,10 +1444,7 @@ def norm(self, x):
         norm : `float`
             The norm of the vector
         """
-        if self.exponent == 2.0:
-            from math import sqrt
-            return sqrt(self.const) * float(_norm_default(x))
-        elif self.exponent == float('inf'):  # Weighting irrelevant
+        if self.exponent == float('inf'):  # Weighting irrelevant
```

Comment:
No reason (imo) to single out the exponent == 2 case. If someone uses this one we can kinda assume it is not 2, and if it is, the performance loss is minimal.